src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/BlackDuckVulnerabilityCollector.java

```diff
@@ -176,4 +182,19 @@ private LinkableItem getSeverity(String vulnerabilityUrl) {
         return severityItem;
     }
 
+    private Optional<String> getCvss3Severity(VulnerabilityView vulnerabilityView) {
+        Boolean useCvss3 = vulnerabilityView.getUseCvss3();
+        if (null != useCvss3 && useCvss3) {
+            JsonObject vulnJsonObject = vulnerabilityView.getJsonElement().getAsJsonObject();
```

Comment:
This looks fine to me. Could we add a TODO to replace this when blackduck-common-api is updated to expose the cvss3 and its severity as Java objects?